spec/consumer_spec.rb

```diff
@@ -1,11 +1,12 @@
 require_relative 'spec_helper'
 
 describe Consumer do
-  let(:message_values) { [:delivery_info1, :headers1, "message1_payload"] }
+  let(:yield_values) { [:delivery_info1, :headers1, "message1_payload"] }
```

Comment:
Yeah, inlining here will be better. Argument order differs on what the message receives and what the subscription block provides.